examples/nat_scapy_tests.py

```diff
@@ -144,9 +144,11 @@ def run_server2(serverport=12022, natport=35002):
     ackn += 1
     print "Replying with Ack"
     send(ip/TCP(sport=serverport,dport=natport,flags="A", seq=seqn,ack=ackn))
+    time.sleep(1)
```

Comment:
Yeah sorry, I tend to end up describing things in a very contextual dependent manner, even when that context is only in my head!

The second scapy test is supposed to fail if we can still send data across the NAT after we have closed the connection with Fin. The test should have been failing because I had added TIME_WAIT support, but not adjusted the test, so the packets would have gotten through.

The scapy tests are very fragile because we sniff packets instead of getting them from a queue (meaning we need to make allowances for timing), and we don't really check that the packets are actually what we want. I would hope to replace them or rewrite them eventually, but since they work for now I think it better to focus on improving the NAT.